packages/headless/src/controllers/index.ts

```diff
@@ -218,6 +216,7 @@ export {
   CartRecommendationsListOptions,
   CartRecommendationsListProps,
   CartRecommendationsListState,
+  ProductRecommendation,
```

Comment:
@francoislg, I felt `CartProductRecommendation` would be a more specific name to export. At the same time, I was not sure if other controllers could one day expose `ProductRecommendation` objects? Looking at the reducers, I also saw general language.

Any thoughts around this? Would a general name be best?

Reply:
So far, the plan is for all recommendations to use the same format